Assets/Thirdweb/Examples/Scripts/Prefab_ConnectWallet.cs

```diff
@@ -170,66 +137,14 @@ public void OnTryConnect(string _walletStr)
 
     public void OnClickDropdown()
     {
-        dropdownExpand = !dropdownExpand;
-
-        if (dropdownRoutine != null)
-            StopCoroutine(dropdownRoutine);
-
-        dropdownRoutine = StartCoroutine(DropdownRoutine(String.IsNullOrEmpty(address) ? connectDropdown : connectedDropdown));
-    }
-
-    IEnumerator DropdownRoutine(Transform _object)
-    {
-        _object.gameObject.SetActive(true);
-        Vector3 targetScale = dropdownExpand ? Vector3.one : Vector3.zero;
-        while (Math.Abs(_object.localScale.x - targetScale.x) > 0.01f)
-        {
-            _object.localScale = Vector3.Lerp(_object.localScale, targetScale, 10f * Time.deltaTime);
-            yield return new WaitForEndOfFrame();
-        }
-        _object.localScale = targetScale;
-        _object.gameObject.SetActive(dropdownExpand);
+        if (String.IsNullOrEmpty(address))
+            connectDropdown.SetActive(!connectDropdown.activeInHierarchy);
+        else
+            connectedDropdown.SetActive(!connectedDropdown.activeInHierarchy);
     }
 
     // Utility
 
-    int GetChainID(Chain _chain)
```

Comment:
wonder if it's worth leaving this but using string (could also help validate the inputted string)

just until we ship the new chain id SDK input